pkg/operator/controller/status.go

```diff
@@ -53,6 +53,18 @@ func (r *reconciler) syncOperatorStatus() error {
 	if err != nil {
 		return fmt.Errorf("failed to get operator state: %v", err)
 	}
+
+	related := defaultRelatedObjects(ns.Name)
+	for _, ingress := range ingresses {
+		related = append(related, configv1.ObjectReference{
+			Group:     operatorv1.GroupName,
+			Resource:  "IngressController",
+			Namespace: r.Namespace,
```

Comment:
Semantically it might make more sense to use `ingress.Namespace` here.  Admittedly, it makes no practical difference.

Reply:
Yeah... file this under namespace handling tech debt...